src/prefect/tasks/mysql/mysql.py

```diff
@@ -158,6 +162,7 @@ def run(
             - query (str, optional): query to execute against database
             - commit (bool, optional): set to True to commit transaction, defaults to false
             - charset (str, optional): charset of the query, defaults to "utf8mb4"
+            - cursor_type (Union[str, Callable], optional): cursor type you want to use (defaults to standard Cursor class)
```

Comment:
```suggestion
            - cursor_type (Union[str, Cursor], optional): cursor type you want to use (defaults to standard Cursor class)
```

Reply:
This type signature isn't correct, you don't pass in a `Cursor` type, you pass in a callable that might create a cursor type. IMO the `Callable` signature was fine.

Reply:
Ah yep my bad, I definitely conflated this during the review. I think (for annotation purposes) `Type[Cursor]` is more informative to users than `Callable` unless you want to extend the `Callable` annotation to indicate a return type of `Cursor` but however this is done I think it's not a big deal and not worth getting further into the weeds -- I'll be happy with whatever is submitted.